apps/agreement/contracts/Agreement.sol

```diff
@@ -1125,8 +1125,7 @@ contract Agreement is IAgreement, AragonApp {
             return false;
         }
 
-        // TODO: update with new ACL version: no need to pass challenger address by parameter
-        bytes memory params = ConversionHelpers.dangerouslyCastUintArrayToBytes(arr(_challenger));
+        bytes memory params = new bytes(0);
```

Comment:
We can probably inline this one now :)

Reply:
Sure! I was about to do it, but I thought you liked this style :sweat_smile: